script/export_for_foi/export.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+timestamp=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+echo 'Started site export'
+psql -d transition_development -f ./export_sites.sql > ./sites-$timestamp.csv
```

Comment:
I think as long as the data in development is recent (a few days old) I think it's totally reasonable to run the export in development.